processing/index.ts

```diff
@@ -0,0 +1,62 @@
+import { $ } from 'bun'
+import { downloadFile, waitForFreshData } from './download'
+import { idFilter, tagFilter } from './filter'
+import { parseParameters } from './utils/parameters'
+import { processTopics } from './processTopics'
+import {
+  HASHES_DIR,
+  FILTER_DIR,
+  ID_FILTERED_FILE,
+  OSM_DOWNLOAD_DIR,
+  OSM_FILTER_DIR,
+} from './directories.const'
+import { TopicList } from './topics.const'
+import { directoryHasChanged, updateDirectoryHash } from './utils/hashing'
+import { writeMetadata } from './metadata'
+
+const params = parseParameters()
+
+$`mkdir -p ${OSM_DOWNLOAD_DIR} ${OSM_FILTER_DIR} ${HASHES_DIR}`
+
+if (params.waitForFreshData) {
+  await waitForFreshData(params.fileURL, 24, 10)
+}
+const { fileName, fileChanged } = await downloadFile(params.fileURL, params.skipDownload)
+
+if (fileChanged && !(await directoryHasChanged(FILTER_DIR))) {
+  await tagFilter(fileName)
+  updateDirectoryHash(FILTER_DIR)
+}
+
+if (params.idFilter && params.idFilter !== '') {
+  await idFilter(fileName, params.idFilter)
+  await processTopics(TopicList, ID_FILTERED_FILE, fileChanged, params)
+} else {
+  await processTopics(TopicList, fileName, fileChanged, params)
+}
+
+await writeMetadata(fileName, 0)
+
+// restart `tiles` container
```

Comment:
```suggestion
// restart `tiles` container
// … because
```

Warum machen wir das nochmal an dieser Stelle?

Reply:
Damit die neuen Tables von atlas erkannt werden, habe aber gerade gemerkt, dass es eigentlich nach der post-processing-hook passieren sollte, damit die evtl. neuen functions auch geupdatet werden.